Code/GraphMol/ChemReactions/SanitizeRxn.h

```diff
@@ -82,7 +82,22 @@ inline const MolOps::AdjustQueryParameters DefaultRxnAdjustParams() {
 }
 
 // Default adjustment parameters for ChemDraw style matching of reagents
+//  -- deprecated - renamed MatchOnlyAtRgroupsAdjustParams
+//  -- this doesn't match sciquest style searching
 inline const MolOps::AdjustQueryParameters ChemDrawRxnAdjustParams() {
```

Comment:
Missed this in the first pass: something should be added to the release notes for the deprecation and a github issue should probably be created